src/layouts/MainAppLayout/Header/CurrencyButton/styles.js

```diff
@@ -0,0 +1,49 @@
+export const gridProps = (backgroundImage) => ({
```

Comment:
You can define a single object which you export and nest these two objects in it

```
const styles = {
gridProps: {},
buttonProps: {}
}

export default styles

```

Then use it like : 

`
<Grid {...styles.buttonProps} />
`